css/style.css

```diff
@@ -15,10 +15,31 @@ h1, h2, h3, h4 {
   text-transform: uppercase;
 }
 
+.row {
```

Comment:
We might use rows in other places of the template as well, so I would specify this a bit more. Which row(s) exactly should this apply to?